web/react/components/post_list_container.jsx

```diff
@@ -2,7 +2,14 @@
 // See License.txt for license information.
 
 const PostList = require('./post_list.jsx');
+const TutorialIntroScreens = require('./tutorial/tutorial_intro_screens.jsx');
```

Comment:
Code in this file should probably be moved to ChannelView after my changes go in.